pkg/registry/apis/datasource/register.go

```diff
@@ -167,9 +197,9 @@ func getHardcodedQueryTypes(group string) (*queryV0.QueryTypeDefinitionList, err
 	var err error
 	var raw json.RawMessage
 	switch group {
-	case "testdata.datasource.grafana.app":
+	case "testdata.datasource.grafana.app", "testdata":
```

Comment:
```suggestion
	case "testdata.datasource.grafana.app", "grafana-testdata-datasource":
```
The plugin ID is https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/grafana-testdata-datasource/plugin.json#L4

"testdata" is a alias... ideally we can support that as a redirect, but I don't think it should have its own apiserver